package.json

```diff
@@ -13,10 +13,12 @@
     "babel": "^5.8.20",
     "css-modules-loader-core": "0.0.12",
     "in-publish": "^2.0.0",
+    "isparta": "^3.0.3",
     "mocha": "^2.2.5"
   },
   "scripts": {
-    "test": "npm run -s build && mocha --compilers js:babel/register",
+    "test": "mocha --compilers js:babel/register",
+    "test:cov": "`npm bin`/babel-node `npm bin`/isparta cover --report text --report html `npm bin`/_mocha",
```

Comment:
Here are couple of tricks with babel-node, so nothing fails on tests written in ES6